assets/logo.inkscape.svg

```diff
@@ -1,50 +1,131 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg
+   xmlns:osb="http://www.openswatchbook.org/uri/2009/osb"
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:cc="http://creativecommons.org/ns#"
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:svg="http://www.w3.org/2000/svg"
    xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   sodipodi:docname="wikijump2.svg"
+   sodipodi:docname="logo.inkscape.svg"
    inkscape:version="1.0 (4035a4fb49, 2020-05-01)"
-   inkscape:export-ydpi="96"
-   inkscape:export-xdpi="96"
-   inkscape:export-filename="C:\Users\Ross\Documents\wikijump2.png"
    id="svg8"
    version="1.1"
-   viewBox="0 0 58.264759 16.036171"
-   height="16.036171mm"
-   width="58.264759mm">
+   viewBox="0 0 569.79474 141.29433"
+   height="141.29433mm"
+   width="569.79474mm"
+   inkscape:export-filename="C:\Users\Ross\Downloads\wikijump.png"
```

Comment:
This meta field should be removed.

Reply:
Good point - there's probably a lot of metadata to strip. I wonder if there's a way to stop Inkscape from adding it?

Not much point removing it until we're on the precipice of merging, but definitely good to keep in mind for then